### PR TITLE
Set up a "Hello Microverse" project (first project)

### PR DESCRIPTION
**Pull request by omar25ahmed**
**First project:**
- I have created html file and linked it to css sheat also 
- I have created a git ignore file  
- all files are linked to a linter that defines the problem with code and if there was a bug that has to be fixed 
- read me file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (MrBrN197) [inline .gitignore]: - [x] Hello good work adding the test.md file. Please also include the node_modules/ folder in the gitignore file.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi,  @omar25ahmed 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] You've created a pull request description that is not correct. 
Your pull request should not be a message to ask someone to review your code. 
it should be a description of what you have implemented this branch.
It's a **description** of the pull request

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — APPROVED** [review]

**Comment by omar25ahmed** [comment]
so is there any changes need to be done Brian?
I need more clarification please!

File: .gitignore (hunk 1 above)
Comment (omar25ahmed) [reply .gitignore]: can you please give me more clarification?

**Comment by MrBrN197** [comment]
> so is there any changes need to be done Brian?
> I need more clarification please!

@omar25ahmed Yes they are. please ignore the "approved" comment that was a mistake on my part. sorry. And I am unable to remove it. please still implement the changes I asked for and ask for a re-review

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @omar25ahmed, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- _Please make your pull request title descriptive, telling you what changes are inside.._
- _Your pull request should be a short summary describing introduced changes in more detail. Please check this [LINK](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) on how to write a descriptive title and a short summary_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+node_modules/
```
Comment (mateo951) [inline .gitignore:2]: Nice work implementing the required changes by the previous reviewer 🙌🔥

**Review by mateo951 — APPROVED** [review]
Hi @omar25ahmed,

You have a really good project here! Congrats 😎🙌 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

#### Be sure to check the following [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) when you write your next PR 🧐 This will give you a better idea on how to improve even more your PR title and summary. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**